plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -915,8 +912,9 @@ def create_vm(module, proxmox, vmid, newid, node, name, memory, cpu, cores, sock
         if 'pool' in kwargs:
             del kwargs['pool']
 
-    # Convert all dict in kwargs to elements. For hostpci[n], ide[n], net[n], numa[n], parallel[n], sata[n], scsi[n], serial[n], virtio[n], ipconfig[n]
-    for k in list(kwargs.keys()):
+    # Convert all dict in kwargs to elements.
+    # For hostpci[n], ide[n], net[n], numa[n], parallel[n], sata[n], scsi[n], serial[n], virtio[n], ipconfig[n]
+    for k in kwargs.keys():
```

Comment:
This is a breaking change. `kwargs` is modified during iteration.

Reply:
oopsie, thanks for catching.